### PR TITLE
Сode review

### PR DESCRIPTION
**Pull request by AlexeyBondziuk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,27 +12,27 @@
...
-                    <li  class="header__item"><a class="header__href" href="">Правообладателям</a></li>
-                    <li  class="header__item"><a class="header__href" href="">Заметили ошибку? <span class="header__href_hidden">Сообщите нам!</span> </a></li>
-                    <li  class="header__item"><a class="header__href" href="">Стол заказов</a></li>
-                    <li  class="header__item"><a class="header__href" href="">Мы Вконтакте | Твиттере</a></li></ul>
+            <div>
+                <div class="burger">
+                    <a class="burger__link" href="">
+                        Главное меню
+                    </a>
                 </div>
```
Comment (AlexeyBondziuk) [inline index.html:20]: I just have fixed formatting with spaces

File: index.html [hunk 2]
```diff
@@ -12,27 +12,27 @@
...
-                            <input aria-label="Найти фильм" class="header__search" type="text" name="search reqest">
-                            <button class="header__btn">Поиск</button>                         
-                        </form>
-                        <button class="header__btn header__btn_less310px">Найти фильм</button>
-                    </div>
-            </nav>
+            <ul class="header__list">
+                <li class="header__item"><a class="header__href"  href="">Главная</a></li>
+                <li  class="header__item"><a class="header__href" href="">Правообладателям</a></li>
+                <li  class="header__item"><a class="header__href" href="">Заметили ошибку? <span class="header__href_hidden">Сообщите нам!</span> </a></li>
```
Comment (AlexeyBondziuk) [inline index.html:24]: The length of this line is too big, best practise is to use 110-130 symbols in one line

File: index.html [hunk 3]
```diff
@@ -12,27 +12,27 @@
...
+                <li  class="header__item"><a class="header__href" href="">Стол заказов</a></li>
+                <li  class="header__item"><a class="header__href" href="">Мы Вконтакте | Твиттере</a></li></ul>
+            </div>
+                <div class="header__right">
+                    <form class="header__form" action="https://jsonplaceholder.typicode.com/posts" method="post">
```
Comment (AlexeyBondziuk) [inline index.html:29]:
```suggestion
                    <form 
                        class="header__form" 
                        action="https://jsonplaceholder.typicode.com/posts" 
                        method="post"
                    >
```

Also is just suggestion in formating, it is better to put atributes one under another. It is esier to read

File: index.html [hunk 4]
```diff
@@ -12,27 +12,27 @@
...
+                        <input aria-label="Найти фильм" class="header__search" type="text" name="search reqest">
+                        <button class="header__btn">Поиск</button>
+                    </form>
+                    <button class="header__btn header__btn_less310px">Найти фильм</button>
+                </div>
```
Comment (AlexeyBondziuk) [inline index.html:34]:
```suggestion
            <div class="header__right">
                <form class="header__form" action="https://jsonplaceholder.typicode.com/posts" method="post">
                    <input aria-label="Найти фильм" class="header__search" type="text" name="search reqest">
                    <button class="header__btn">Поиск</button>
                </form>
                <button class="header__btn header__btn_less310px">Найти фильм</button>
            </div>
```

File: index.html (hunk 1 above)
Comment (SlaVR7) [reply index.html:20]: ok, it's good fix

**Comment by SlaVR7** [comment]
Not bad, but i wanna create my own pull reqest

**Pull request closed without merging**